cardinal-components-entity/src/main/java/dev/onyxstudios/cca/mixin/entity/common/MixinEntity.java

```diff
@@ -107,5 +108,9 @@ public ComponentContainer getComponentContainer() {
         writer.writeSyncPacket(buf, recipient);
         return new CustomPayloadS2CPacket(CardinalComponentsEntity.PACKET_ID, buf);
     }
-
+    
+    @Inject(method = "onStartedTrackingBy", at = @At("HEAD"))
```

Comment:
This inject location seems a little risky, since it's an empty method there is no guarantee other entities will call `super`